Doc/Tutorial/chapter_testing.tex

```diff
@@ -431,7 +431,7 @@ \section{Writing doctests}
 \label{section:doctest}
 
 Python modules, classes and functions support built in documentation using
-docstrings.  The \href{http://docs.python.org/library/doctest.html}{doctest
+docstrings.  The \href{https://docs.python.org/2/library/doctest.html}{doctest
```

Comment:
--> https://docs.python.org/3/library/doctest.html

Reply:
I use the reference to Python 3 documentation.